FSW/headers/eventUtil.hpp

```diff
@@ -0,0 +1,52 @@
+#ifndef EVENTUTIL_H
+#define EVENTUTIL_H 
+
+/* - - - - - - Includes - - - - - - */
+// C++ libraries
+#include <vector>
+
+// Other libraries
+#include <Arduino.h>
+
+// NS2 headers
+
+
+/* - - - - - - Declarations - - - - - - */
+
+/* - - - - - - Class Definitions - - - - - - */
+
+class Event
```

Comment:
It would help if we write up either some external documentation or write some general code comments in here about what the classes do. I think classes can get confusing when there ends up being a lot to go track.